packages/proto-signing/src/registry.ts

```diff
@@ -94,7 +94,13 @@ export class Registry {
       return this.decodeTxBody(value);
     }
     const type = this.lookupTypeWithError(typeUrl);
-    return type.decode(value);
+    const decoded = type.decode(value);
+    Object.entries(decoded).forEach(([key, val]: [string, any]) => {
+      if (val.constructor.name === "Buffer") {
```

Comment:
We can also use `Buffer.isBuffer`:

```ts
  if (typeof Buffer !== "undefined" && typeof Buffer.isBuffer !== "undefined") {
    // Buffer.isBuffer is available at runtime
    if (Buffer.isBuffer(data)) return false;
  }
```

See isUint8Array

Reply:
Alternatively we can simply do

```ts
if (Object.prototype.toString.call(val) === "[object Uint8Array]") {
  // here val is `Uint8Array` or `Buffer`
  decoded[key] = Uint8Array.from(val);
}
```

Reply:
I like the `Buffer.isBuffer` option as being the closest to what we actually mean.